setup/emc-update-impl.bash

```diff
@@ -118,7 +118,7 @@ if [ "$ROBOT_REAL" = true ] ; then
 fi
 
 # 3) Install dependencies
-_make_sure_installed ros-"$EMC_ROS_DISTRO"-cv-bridge ros-"$EMC_ROS_DISTRO"-image-geometry ros-"$EMC_ROS_DISTRO"-tf libassimp-dev ros-"$EMC_ROS_DISTRO"-message-runtime ros-"$EMC_ROS_DISTRO"-message-generation ros-"$EMC_ROS_DISTRO"-roscpp ros-"$EMC_ROS_DISTRO"-nav-msgs ros-"$EMC_ROS_DISTRO"-shape-msgs
+_make_sure_installed ros-"$EMC_ROS_DISTRO"-cv-bridge ros-"$EMC_ROS_DISTRO"-map-server ros-"$EMC_ROS_DISTRO"-image-geometry ros-"$EMC_ROS_DISTRO"-tf libassimp-dev ros-"$EMC_ROS_DISTRO"-message-runtime ros-"$EMC_ROS_DISTRO"-message-generation ros-"$EMC_ROS_DISTRO"-roscpp ros-"$EMC_ROS_DISTRO"-nav-msgs ros-"$EMC_ROS_DISTRO"-shape-msgs
```

Comment:
```suggestion
_make_sure_installed libassimp-dev ros-"${EMC_ROS_DISTRO}"-cv-bridge ros-"${EMC_ROS_DISTRO}"-image-geometry ros-"${EMC_ROS_DISTRO}"-map-server ros-"${EMC_ROS_DISTRO}"-message-generation ros-"${EMC_ROS_DISTRO}"-message-runtime ros-"${EMC_ROS_DISTRO}"-nav-msgs ros-"${EMC_ROS_DISTRO}"-roscpp ros-"${EMC_ROS_DISTRO}"-shape-msgs ros-"${EMC_ROS_DISTRO}"-tf
```
Update this one we replacing `tf` by `tf2`

Reply:
```suggestion
_make_sure_installed libassimp-dev ros-"${EMC_ROS_DISTRO}"-cv-bridge ros-"${EMC_ROS_DISTRO}"-image-geometry ros-"${EMC_ROS_DISTRO}"-map-server ros-"${EMC_ROS_DISTRO}"-message-generation ros-"${EMC_ROS_DISTRO}"-message-runtime ros-"${EMC_ROS_DISTRO}"-nav-msgs ros-"${EMC_ROS_DISTRO}"-roscpp ros-"${EMC_ROS_DISTRO}"-shape-msgs ros-"${EMC_ROS_DISTRO}"-tf2
```

To be sure, _ros-"${EMC_ROS_DISTRO}"-tf_ needs to be changed to _ros-"${EMC_ROS_DISTRO}"-tf2_ ?

Reply:
last `tf`usage:
https://github.com/tue-robotics/emc_simulator/blob/c8c7d7eea1427095c03987f8f9df47f85f14334c/src/virtualbase.h#L139

Reply:
Removed